controllers/authController.js

```diff
@@ -0,0 +1,91 @@
+const User = require('./../models/User.model')
+const bcryptjs = require('bcryptjs')
+exports.viewRegister = (req, res) => {
+      res.render('auth/signup')
+}
+
+exports.register = async (req, res) => {
+      const username = req.body.username
```

Comment:
con las siguente destructuracion podemos reducir las lineas de codigo cuando tienes muchas propiedades 😉 
```suggestion
      const {username, password} = req.body
```